src/shared/hooks/useTable.ts

```diff
@@ -17,32 +17,29 @@ export const useTable = (
 
   const [searchValue, setSearchValue] = useState('');
   const [page, setPage] = useState(1);
-  const [orderBy, setOrderBy] = useState<OrderBy>(OrderBy.UpdatedAt);
+  const [orderBy, setOrderBy] = useState('');
   const [order, setOrder] = useState<Order>('desc');
+  const ordering = orderBy ? `${order === 'asc' ? '+' : '-'}${orderBy}` : '';
```

Comment:
make a enum from order type to replace 'asc' to Order.asc
and maybe move this functions to utils